robin_stocks/robinhood/helper.py

```diff
@@ -349,6 +349,8 @@ def request_post(url, payload=None, timeout=16, json=False, jsonify_data=True):
                 'Content-Type', 'application/x-www-form-urlencoded; charset=utf-8')
         else:
             res = SESSION.post(url, data=payload, timeout=timeout)
+        if res.status_code in [500,400]:
```

Comment:
More of a code styling thing but I would do [400, 500] as it logically makes sense